centos8-stream/README.md

```diff
@@ -0,0 +1,62 @@
+# CentOS 8 Stream Packer Template for MAAS
+
+## Introduction
+The Packer template in this directory creates a CentOS 8 Stream AMD64 image for use
+with MAAS.
+
+## Prerequisites (to create the image)
+
+* A machine running Ubuntu 18.04+ with the ability to run KVM virtual machines.
+* qemu-utils
+* [Packer.](https://www.packer.io/intro/getting-started/install.html)
+
+## Requirements (to deploy the image)
+
+* [MAAS](https://maas.io) 2.3+, [MAAS](https://maas.io) 2.7+ recommended
```

Comment:
Bump to MAAS 3.0 (oldest version currently supported) and 3.2 (current stable version)